go-controller/pkg/ovn/ovn.go

```diff
@@ -3,6 +3,7 @@ package ovn
 import (
 	"bytes"
 	"fmt"
+	nettypes "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
```

Comment:
move this into a  different import block. This first one is for built ins, second one is for this projects imports.

Reply:
done